modules/core/src/internal/get_uv_attribute.cpp

```diff
@@ -0,0 +1,96 @@
+#include <lagrange/Attribute.h>
```

Comment:
Can you add the missing header on this file? Thanks!

Reply:
done, thanks for catching that

Reply:
Don't forget to backport it as well!